### PR TITLE
Updating with GitHub for Python and Gradle

### PR DESCRIPTION
**Pull request by tkadlec**

### CODE REVIEW

**Comment by tkadlec** [comment]
@darscan Can you have a look and see if there are any changes I didn't cover here?

File: _docs_github/4_fix_vulnerabilities_web_flow.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ generate a Snyk pull request that fixes only this vulnerability.
 
 You can review the vulnerabilities that will be fixed, change your selection, and choose to ignore any vulnerabilties that can't be fixed right now before opening the pull request on the 'Open a fix PR' page.
 
-Note that patching is only supported for Node.js repos; Ruby vulnerabilities can be fixed with upgrades only.
+Note that patching is only supported for Node.js repos; Ruby, Python and Java vulnerabilities can be fixed with upgrades only.
```
Comment (darscan) [inline _docs_github/4_fix_vulnerabilities_web_flow.md]: I'd revert this line... Python and Java do not support Fix yet.

File: _docs_python/1_support.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ---
 title: "What type of Python projects does Snyk support?"
 ---
-Our Snyk [CLI](/docs/using-snyk/) tool supports testing and monitoring Python projects that have their dependencies managed by [pip](https://pypi.python.org/pypi/pip) and expressed in a `requirements.txt` file.
+Snyk supports testing and monitoring supports testing and monitoring Python projects that have their dependencies managed by [pip](https://pypi.python.org/pypi/pip) and expressed in a `requirements.txt` file. Python support is available for [GitHub repositories](/docs/github), and via the Snyk [CLI](/docs/using-snyk/).
```
Comment (darscan) [inline _docs_python/1_support.md]: `Snyk supports testing and monitoring supports testing and monitoring` .. whoops

Comment (tkadlec) [inline _docs_python/1_support.md]: It was so nice we had to say it twice

**Pull request merged**